Survey/Survey/Resources/Images.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Images.swift
+//  Survey
+//
+//  Created by Taher on 27/10/22.
+//
+
+import Foundation
+import SwiftUI
+
+enum Images {
+
+    static let logoWhite: String = "LogoWhite"
+    static let splashBackground: String = "SplashBackground"
+    static let overlay: String = "Overlay"
+}
```

Comment:
To avoid dealing with the literal string for resources (for now they're just images, but later you will need to add custom fonts, colors, localized strings, ...), please consider integrating one of these tools: [R.swift](https://github.com/mac-cain13/R.swift) or [SwiftGen](https://github.com/SwiftGen/SwiftGen) 🙏

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/38/commits/ed946222cb171779b44bd26dcc147c0fbc9ed649)